lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,127 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+// \returns true if all dimensions of the input tensors are the same except
```

Comment:
very minor: can you make all these comments Doxygen comments: '///'

Reply:
thx for pointing out! Will correct these in next PR.